breathecode/mentorship/serializers.py

```diff
@@ -485,13 +535,50 @@ class Meta:
         exclude = ('created_at', 'updated_at', 'user', 'token')
 
     def validate(self, data):
+        lang = data.get('lang', 'en')
         if 'status' in data and data['status'] in ['ACTIVE', 'UNLISTED'
                                                    ] and self.instance.status != data['status']:
             try:
                 actions.mentor_is_ready(self.instance)
             except Exception as e:
                 raise ValidationException(str(e))
 
+        user = data['user'] if 'user' in data else self.instance.user
+        academy = data['academy'] if 'academy' in data else self.instance.academy
+        profile_academy = ProfileAcademy.objects.filter(user__id=user.id, academy=academy).first()
+
+        if 'name' not in data:
+            data['name'] = ''
+
+        if not data['name'] and profile_academy:
+
+            data['name'] = profile_academy.first_name + ' ' + profile_academy.last_name
```

Comment:
`' '` is considered as True

Reply:
there is data['name'].strip to prevent that from happenning